pkg/koordlet/runtimehooks/reconciler/reconciler.go

```diff
@@ -169,10 +309,16 @@ func (c *reconciler) reconcilePodCgroup(stopCh <-chan struct{}) {
 					}
 				}
 				for _, containerStat := range podMeta.Pod.Status.ContainerStatuses {
-					for _, r := range globalCgroupReconciler[ContainerLevel] {
-						containerCtx := protocol.HooksProtocolBuilder.Container(
-							podMeta, containerStat.Name)
-						if err := r.fn(containerCtx); err != nil {
+					for _, r := range globalCgroupReconcilers.containerLevel {
+						reconcileFn, ok := r.fn[r.filter.Filter(podMeta)]
+						if !ok {
+							klog.Warningf("calling reconcile function %v failed, error condition %s not registered",
```

Comment:
this warning log should be ignored, e.g. bvt is only registered on pod/kubeqos level

Reply:
fixed